src/main/java/Parser.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Parses user input.
+ */
+
+public class Parser {
+    
+    public Parser() {
+    }
+
+    /**
+     * Parses user input into command for execution.
+     * @param command user string input
+     * @return the command based on string input
+     */
+    public Command parse(String command) {
+        if (command.startsWith("todo ")) {
+            ToDoCommand c = new ToDoCommand(command);
+            return c;
```

Comment:
Could place `return c` at the end of the if-else block instead of repeating them after each case.